client/src/components/Footer.js

```diff
@@ -1,34 +1,39 @@
-import React from "react";
+import React from 'react';
+import '../styles/footer.css';
 
 const Footer = () => {
   return (
     <div className="footer">
-      <div className="col">
+      <div className="footer-col">
         <ul>
+          <h5 className="footer-header">Header</h5>
```

Comment:
I know this isn't your code originally, but we should fix this. Why does the footer have many columns like this...? I don't understand. @simonhlee97 ?

Reply:
This was not my naming convention. I still had a question on your future plans for the footer. I can't continue styling the footer till I have an idea of what you plan on putting there